mozfun/norm/metadata/udf.sql

```diff
@@ -16,34 +16,61 @@ CREATE OR REPLACE FUNCTION norm.metadata(metadata ANY TYPE) AS (
               -- Even though it's not part of the spec, many clients append
               -- '+00:00' to their Date headers, so we strip that suffix.
               REPLACE(metadata.header.`date`, 'GMT+00:00', 'GMT')
-            ) AS parsed_date
+            ) AS parsed_date,
+            ARRAY(
+              SELECT
+                TRIM(t)
+              FROM
+                UNNEST(SPLIT(metadata.header.x_source_tags, ',')) t
+            ) AS parsed_source_tags
```

Comment:
I'm open to feedback on naming here. This could be `parsed_x_source_tags`.

Reply:
`parsed_x_source_tags` might be better, since it makes it more obvious what it is derived from.